### DISCUSSION TITLE
where do you set the preset threshold values (green / yellow / red) ?

### DISCUSSION THREAD

**Discussion by bart3r** [cybertec-postgresql/pgwatch2#453] [Q&A] [answered]
I cannot see where they are set for each metric
Any help?

**Comment by markwort** [comment, marked as answer]
do you mean this?
![image](https://user-images.githubusercontent.com/38984608/154870407-aabc2431-dcac-474c-a101-ffeda8427db9.png)

That is accessible if you choose a graph in grafana, then press `edit` and then it should be on the right in this panel.
I don't think there are predefined values in all diagrams.

If this does not solve what you're looking for, perhaps you can show a screenshot of what you're looking for?

best regards
Julian
